Library/ViewModels/MessageBannerViewModel.swift

```diff
@@ -53,14 +54,34 @@ MessageBannerViewModelInputs, MessageBannerViewModelOutputs {
     self.messageTextAlignment = bannerType.map { $0.textAlignment }
 
     self.bannerMessage = self.messageBannerConfiguration.signal.skipNil().map(second)
+    self.bannerMessageAccessibilityLabel = self.bannerMessage
+      // todo: Needs to be localized - Strings.banner_tap_to_dismiss
+      .map { "\($0), double tap to dismiss" }
```

Comment:
Has this been added to our strings yet?

I think it's always a good idea to use the `localizedString()` function directly if we don't yet have it in `Strings.swift` that way if we forget about this it's going to use localization or if we need an easy way to find it later we can just search for uses of `localizedString()`.

Reply:
I completely forgot about this...good catch Justin.

Created https://github.com/kickstarter/kickstarter/pull/14146 and will block this PR until we have those strings.

Reply:
Also I had to update the string to be in form of a sentence to better match server response..

Before
<img width="1025" alt="screen shot 2019-01-31 at 11 19 02 am" src="https://user-images.githubusercontent.com/387596/52080111-e702b500-254b-11e9-94aa-0d37fc6d9574.png">

After
<img width="1025" alt="screen shot 2019-01-31 at 11 27 03 am" src="https://user-images.githubusercontent.com/387596/52080134-ee29c300-254b-11e9-9b79-46a4fc9196c1.png">

Reply:
Also note that we should remove `project_count_projects_backed` when https://github.com/kickstarter/kickstarter/pull/13979#pullrequestreview-198853575 gets merged